src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -10,6 +11,7 @@
 public class MovieServiceImpl implements MovieService {
     @Inject
     private MovieDao movieDao;
+    private Throwable throwable;
```

Comment:
do we need this field here?

Reply:
)) I don't even know where it comes from.
Canceled